### ISSUE TITLE
load_dotenv()  UnicodeDecodeError

### ISSUE THREAD

**Issue by JasonWuuu** [theskumar/python-dotenv#207]
**in .evn file** 

`
app_key = '201909128757'
account_name = '中国股份有限公司'
`
**in home.py**
`
from dotenv import load_dotenv, find_dotenv
import os

load_dotenv(find_dotenv())

print(os.getenv('account_name'))
`

**error:**

File "C:\Python37\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 447: character maps to <undefined>

**Comment by bbc2** [comment]
I can't reproduce the issue. In what encoding is your .env file and what's the default encoding on your system?  You might want to try the `encoding` parameter of `load_dotenv`.

**Comment by jinzhijie** [comment]
I have the same problem with the errorlog: `UnicodeDecodeError: 'gbk' codec can't decode byte 0x91 in position 12: incomplete multibyte sequence`
And I succeed in solve this by the following code:
``` python
load_dotenv(encoding='utf8')
```

手动传个编码就解决了嘛~

**Comment by methane** [comment]
How about changing default encoding from locale specific to UTF-8?

**Comment by andodet** [comment]
Probably duplicate of #300

**Comment by bbc2** [comment]
Yes, and I think that this particular issue was resolved so let me close it.
